l10n_th_withholding_tax_cert/models/withholding_tax_cert.py

```diff
@@ -68,11 +131,25 @@ class WithholdingTaxCert(models.Model):
         comodel_name="res.partner",
         string="Supplier",
         required=True,
-        readonly=True,
+        compute="_compute_wt_cert_data",
+        store=True,
         states={"draft": [("readonly", False)]},
-        copy=False,
         ondelete="restrict",
     )
+    company_id = fields.Many2one(
+        comodel_name="res.company",
+        string="Main Company",
+        required=True,
+        readonly=True,
+        default=lambda self: self._default_company_id(),
```

Comment:
```suggestion
        default=lambda self: self.env.company,
```
I think it better. What do you think?